cagallo/book-club-api
# Feature/express setup ect

## What Features are Being Added?
- Created `server.js` and initial logic for handling get requests.

## What is Being Refactored?
- N/A

## Where should the reviewer start?
- `server.js` 

## How should this be manually tested?
- Boot up the server and try a couple of the routes to see if they are working correctly.

## What are the relevant tickets?
- Setting up the initial server.

## Any Other Notes/Questions/Problems?
- Let me know if there are any revisions that need to be made. Thanks team!
